package.json

```diff
@@ -57,6 +57,8 @@
     "minimatch": "3.0.3",
     "mz": "2.6.0",
     "node-firefox-connect": "1.2.0",
+    "node-notifier": "4.6.1",
+    "regenerator-runtime": "0.10.1",
```

Comment:
remove this line (regenerator runtime is already included in the dependencies at line 63)